src/Parser/Hunter/BeastMastery/Modules/Spells/DireBeast/DireBeast.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import SPELLS from "common/SPELLS/index";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import { formatNumber } from "common/format";
+import SpellLink from "common/SpellLink";
+
+let COOLDOWN_REDUCTION_MS = 12000;
+
+class DireBeast extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+  };
+
+  DBCasts = 0;
```

Comment:
This should be just `casts`. Please don't include the name of the class in a property name (no `car.carSpeed`, just `speed` would be enough).